Header header
float64 north
float64 east
uint8 regular_detections
uint8 priority_detections